### PR TITLE
Reduce number of inputs

### PR DESCRIPTION
**Pull request by Priya2698**
Adding `NVFUSER_DISABLE=kernel_reuse` increases the resource usage by 3x. Reducing the number of inputs (by 4x)  to avoid timeout on CI.

### CODE REVIEW

**Comment by xwang233** [comment]
!build --pybench

**Comment by xwang233** [comment]
!build --pybench

File: benchmarks/python/global_params.py [hunk 1]
```diff
@@ -73,7 +73,7 @@ def generate_input_sizes(dims: Union[int, List] = 2) -> List[Tuple]:
         if dim == 2:
             input_ranges = []
 
-            step_size = 256
+            step_size = 1024
```
Comment (naoyam) [inline benchmarks/python/global_params.py:76]: IIUC, this step size means the problem sizes would be `x + 1024 * i`, where `x` is the minimum size and `i` is increased from 0 until `x + 1024 * i` reaches the maximum size. Is that correct?

Another way of generating problem sizes would be `x + some_factor ^ i`, e.g., `2, 4, 8, 16, 32, ...`.

**Comment by Priya2698** [comment]
Closing as not planned.

**Pull request closed without merging**